### PR TITLE
chore(connect-explorer): gh-action to sync with gh-pages

### PR DESCRIPTION
**Pull request by mroz22**
trezor-connect has on its index https://connect.trezor.io/8/ link to connect-explorer hosted on github pages. https://trezor.github.io/connect-explorer/#/ but this connect-explorer build is fairly outdated. We would like to update connect explorer hosted in github-pages it with a new build with every pipeline on develop branch.

could you please have a look @vdovhanych if this is doable when you have few spare moments?

### CODE REVIEW

File: .github/workflows/connect_explorer_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Build and deploy connect explorer
+        run: |
+          npm install -g yarn && yarn install
+          git config --global user.name "trezor-ci"
+          git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
+          yarn build:libs
+          TREZOR_CONNECT_SRC=https://connect.trezor.io/8/ yarn workspace @trezor/connect-explorer build
```
Comment (szymonlesisz) [inline .github/workflows/connect_explorer_deploy.yml:31]: is `TREZOR_CONNECT_SRC` required? this should be a default, no?

File: .github/workflows/connect_explorer_deploy.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          cd ../connect-explorer-ghpages
+          git checkout gh-pages
+          cp -r ../trezor-suite/packages/connect-explorer/build/* ./
+          git add .
+          git commit -m "new  build"
```
Comment (szymonlesisz) [inline .github/workflows/connect_explorer_deploy.yml]:
```suggestion
          git commit -m "new build"
```

**Comment by vdovhanych** [comment]
would that be deployed from suite repo? I'm not really sure about GitHub pages for this and especially from trezor suite. It's possible but I would use a different workflow for this, will modify it if we decide to deploy it here.

**Comment by tsusanka** [comment]
What's the plan for the connect-explorer repo? Do we have any use for it besides github pages?

**Comment by mroz22** [comment]
To sum up:
- we have an outdated link to connect-explorer in https://connect.trezor.io/8/ 
- we are not sure whether we want to host public connect-explorer on trezor.io domain, because this is a 'trusted' domain and it does not behave in the same way it would behave for 3rd party implementator
- long term we probably want connect-explorer to become index of trezor-connect. still some questions though
- we don't want to do any updates to trezor/connect-explorer repo. maybe apart from gh-pages branch. otherwise this is a dead repo.

File: .github/workflows/connect_explorer_deploy.yml (hunk 1 above)
Comment (mroz22) [reply .github/workflows/connect_explorer_deploy.yml:31]: right now, default is set to `'https://localhost:8088/` unless there is env var specified.

**Pull request closed without merging**